src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs

```diff
@@ -19,15 +19,18 @@ public CosmosReviewRepository(IConfiguration configuration)
             _reviewsContainer = client.GetContainer("APIView", "Reviews");
         }
 
-        public async Task<IEnumerable<ReviewModel>> GetReviewsAsync(bool closed, string language)
+        public async Task<IEnumerable<ReviewModel>> GetReviewsAsync(bool closed, string language, string packageName = "", bool locked = false)
```

Comment:
Consider keeping this method to be used with a review list page and creating a new one for the new purpose. Otherwise, it would get overloaded with too many parameters and become hard to read.